packages/collaboration/src/tokens.ts

```diff
@@ -29,7 +30,23 @@ export const ICurrentUser = new Token<ICurrentUser>(
  * NOTE: Require this token in your extension to access the user menu
  * (top-right menu in JupyterLab's interface).
  */
-export const IUserMenu = new Token<IUserMenu>('@jupyterlab/user:IUserMenu');
+export const IUserMenu = new Token<IUserMenu>(
+  '@jupyterlab/collaboration:IUserMenu'
+);
+
+/**
+ * The user panel token.
+ */
+export const IUserPanel = new Token<AccordionPanel>(
+  '@jupyterlab/collaboration:userPanel'
```

Comment:
For consistency we can rename it to:

```suggestion
  '@jupyterlab/collaboration:IUserPanel'
```